### DISCUSSION TITLE
Could we get an adults version of this list with porn, violence, etc removed?

### DISCUSSION THREAD

**Discussion by pogue** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#635] [Ideas]
I understand your hosts lists is for family viewing. But I am a grown adult and I don't want to be censored and restricting at looking at what I want to look it. However, I still want to be fully protected from malware/tracks/ad servers and so forth.

Is there anyway to talk you into making an adult version of Ultimate.Hosts.Blacklist?

Thank you for taking the time and consider my request.
pogue

**Comment by spirillen** [comment]
Hey @pogue 

I don't know how you are using this list pi-hole/Adblock /etc/hosts etc. but IF you can setup whitelists you I can suggest having a look at my PR project at mypdns.org it is a reversed usage of the intention, but what, known data is known data...

**Comment by CarstenG2** [comment]
just use AdGuard DNS servers for this like 94.140.14.14 + 94.140.15.15 They are made for this. details see here https://adguard.com/en/blog/adguard-dns-new-addresses.html
